docs/pullRequestTemplate.md

```diff
@@ -0,0 +1,25 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# Checklist:
```

Comment:
Is the checklist really suitable for your project? Isn't it too hard?

Where is the entry "Change added to `CHANGELOG.md`"? For CHANGELOG.md see https://keepachangelog.com/ and https://github.com/JabRef/jabref/blob/master/CHANGELOG.md for a real-life example.